Cargo.toml

```diff
@@ -7,7 +7,15 @@ exclude = [".github/"]
 
 
 [workspace]
-members = ["bothan-core", "bothan-binance", "bothan-coingecko", "bothan-kraken", "bothan-api/server", "bothan-api/server-cli"]
+members = [
+    "bothan-api/server",
+    "bothan-api/server-cli",
+    "bothan-binance",
+    "bothan-core",
+    "bothan-coingecko",
```

Comment:
bothan-core should be after bothan-coingecko?

Reply:
so how should we order them, at first I just use your ordering then it is not your liking so I just went with alphabetically.
So please provide me the format you would like.

Reply:
we can fix this in the later source krub

Reply:
it should be alphabetically, "i" comes before "r"

Reply:
oh I remember I put it after, maybe I just blur

Reply:
fixed